.github/workflows/main.yml

```
@@ -96,3 +96,19 @@ jobs:
           ruby-version: "3.3"
           bundler-cache: true
       - run: NO_COVERAGE=true bundle exec rake spec
+
+  prism:
+    runs-on: ubuntu-latest
+    name: Prism
+    steps:
+      - uses: actions/checkout@v4
+      - name: set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          # Specify the minimum Ruby version 2.7 required for Prism to run.
+          ruby-version: 2.7
+          bundler-cache: true
+      - name: spec
+        env:
+          PARSER_ENGINE: parser_prism
+        run: NO_COVERAGE=true bundle exec rake prism_spec
```

```suggestion
        run: NO_COVERAGE=true bundle exec rake
```
What do you think of running all tasks?

---

I apologise for not conveying my idea in detail.
The prism_spec task is the spec task, but run with PARSER_ENGINE=parser_prism
The same env var is already set in the prism ci job, so the spec task there is the same as spec task.

Does this make sense?

---

Uh! I see. Thank you for explaining 🙇🏻 
However, the only difference between prism and parser gems is that they generate different ASTs, so if the spec task is successful, that's enough. Perhaps build_config, internal_investigation, and confirm_config are not different at all.

---

Removed env. How is this?

---

I strongly suggest setting the env var in ci job definition, not in a hacky way shelling out to a rake task with a rake task.

Please correct me if I’m mistaken, but I tend to think this greatly simplifies the whole setup, where the prism job is exactly like others, and no other changes are needed other than the gemfile and gh ci job definitions.

This would also allow us to further isolate the prism job from the parser gem, and regular parser jobs from potential side effects of including the prism gem. Like we do eg for rspec 4 job.

---

But I feel that I’m too pushy and opinionated, so I’d love to hear from @bquorning @Darhazer

---

I see, I agree with your opinion. I think it looks good, but it is not a strong opinion, so I would like @bquorning @Darhazer opinion.